packages/@uppy/companion/Dockerfile.test

```diff
@@ -9,8 +9,7 @@ WORKDIR /app
 
 RUN apk --update add  --virtual native-dep \
   make gcc g++ python libgcc libstdc++ git && \
-  npm install -g corepack && \
-  corepack yarn install && \
+  npm install
```

Comment:
We are not pulling the lock file in the container, let's use npm instead of Yarn since we have to install it anyway.

Reply:
does this solve any problem by using npm install instead of yarn? npm is slower than yarn no?

Reply:
Note sure it's still true when you have to factor in the download+install time from Yarn itself. Anyway, Abdel has improved the PR so it's no longer a concern :)